quickwit/quickwit-indexing/src/metrics.rs

```diff
@@ -66,6 +67,12 @@ impl Default for IndexerMetrics {
                 &[],
                 ["component"],
             ),
+            split_builders_total: new_gauge(
+                "split_builders_total",
```

Comment:
```suggestion
                "split_builders",
```
According to the Prom naming conventions, the `_total` suffix is reserved for accumulating count, i.e., counters. Because of that,  I have a hard time naming gauges, too...

Reply:
oh right!